library/src/main/java/pyxis/uzuki/live/naraeimagepicker/utils/ActivityEx.kt

```diff
@@ -0,0 +1,13 @@
+package pyxis.uzuki.live.naraeimagepicker.utils
+
+import android.support.v7.app.AppCompatActivity
+import pyxis.uzuki.live.naraeimagepicker.item.PickerSettingItem
+
+/**
+ * Apply [PickerSettingItem] theme into Activity if configured.
+ */
+fun AppCompatActivity.applyCustomPickerTheme(settingItem: PickerSettingItem) {
```

Comment:
should be 'internal' modifier cause child module can use this code.

Reply:
@WindSekirun Ok. You mean that we have to restrict that user can't access this function. Am I right?

Reply:
:+1:

Reference: https://kotlinlang.org/docs/reference/visibility-modifiers.html#packages